src/componentManifest.ts

```diff
@@ -51,6 +51,8 @@ interface CompanionManifest extends ComponentManifest {
   settings?: {
     main: string;
   };
+  appClusters?: string[];
+  developerProfileId?: string;
```

Comment:
why have a different name here for `developerProfileId`? I see that at the bottom we pass 
`manifest.developerProfileId = projectConfig.developerID;`. Why not keep `developerID`?